changelog/dev-qit-tests

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: dev
+
+Add script to run QIT security tests locally.
```

Comment:
We don't need to include it in the next release's changelog, do we?

Reply:
I wasn't clear, but I meant it should maybe be added as an "insignificant" changelog, i.e. it should have a `Comment: ` before the text so it's not picked up for the release